Sources/SwiftSyntaxBuilder/FunctionCallExprConvenienceInitializers.swift

```diff
@@ -16,19 +16,33 @@ import SwiftSyntax
 extension FunctionCallExpr {
   public init(
     _ calledExpression: ExpressibleAsExprBuildable,
-    leftParen: TokenSyntax? = .leftParen,
-    rightParen: TokenSyntax? = .rightParen,
+    leftParen: TokenSyntax? = nil,
+    rightParen: TokenSyntax? = nil,
```

Comment:
With the current signature, I would expect that there are no parenthesis if I explicitly pass `nil` to `leftParen` and `rightParen`, even if it results in invalid code.

But since this is a convenience initializer, what do you think about dropping the `leftParen` and `rightParen` arguments altogether and adding a comment to the function that the presence of the parenthesis will be inferred by the presence of arguments?